lib/sync-DataSetModel.js

```diff
@@ -524,44 +524,44 @@ var self = {
     self.globalCollisionRemover = defaultDataHandler.removeCollision;
   },
 
-  toJSON: function (dataset_id, returnData, cb) {
-    var res = {};
-
-    var addData = function (dataset) {
-      for (var i in dataset.clients) {
-        if (dataset.clients.hasOwnProperty(i)) {
-          var dsc = dataset.clients[i];
-          dsc.dataRecords = self.datasetClientRecords[dsc.id];
-        }
-      }
-    }
-
-    if (!dataset_id) {
-      // return entire sync object
-      res = JSON.parse(JSON.stringify(self.datasets));
-      if (returnData) {
-        for (var i in res) {
-          if (res.hasOwnProperty(i)) {
-            var dataset = res[i];
-            addData(dataset);
-          }
-        }
-      }
-      return cb(null, res);
-    }
-    else {
-      self.getDataset(dataset_id, function (err) {
-        if (err) return cb(err);
-
-        if (returnData) {
-          addData(res);
-        }
-        return cb(null, res);
-      });
-    }
-
-
-  },
+  // toJSON: function (dataset_id, returnData, cb) {
+  //   var res = {};
+
+  //   var addData = function (dataset) {
+  //     for (var i in dataset.clients) {
+  //       if (dataset.clients.hasOwnProperty(i)) {
+  //         var dsc = dataset.clients[i];
+  //         dsc.dataRecords = self.datasetClientRecords[dsc.id];
+  //       }
+  //     }
+  //   }
+
+  //   if (!dataset_id) {
+  //     // return entire sync object
+  //     res = JSON.parse(JSON.stringify(self.datasets));
+  //     if (returnData) {
+  //       for (var i in res) {
+  //         if (res.hasOwnProperty(i)) {
+  //           var dataset = res[i];
+  //           addData(dataset);
+  //         }
+  //       }
+  //     }
+  //     return cb(null, res);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)